.flake8

```diff
@@ -0,0 +1,14 @@
+[flake8]
+exclude = docs, __init__.py, versioneer.py
```

Comment:
Similar to UCX-Py, we have a bit of actual code in `__init__.py`. Would there be any downsides of not excluding it from flake8?

Reply:
This comes from here

https://github.com/rapidsai/dask-cuda/blob/2c99f5a0fdf59a7f89e182c4add9121e0b8dd48d/setup.cfg#L10